shared/controller-api/src/main/proto/Controller.proto

```diff
@@ -292,6 +293,15 @@ message DeleteScopeStatus {
     Status status = 1;
 }
 
+message DeleteScopeRecursiveStatus {
```

Comment:
Can we re-use DeleteScopeStatus ?

Reply:
Changes required to re-use the existing Status is completed. Thank you